src/components/graph/Code.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import FileSaver from 'file-saver';
+
+import './Code.css';
+
+class Code extends Component {
+
+  constructor(props){
+    super(props)
+    this.onDownload = this.onDownload.bind(this)
+  }
+
+  onDownload(){
+    let blob = new Blob([this.refs.codeInput.value], {
+       type: "text/plain;charset=utf-8"
+    });
+
+    FileSaver.saveAs(blob, 'synthea_module.json');
```

Comment:
We have a module name, could use it here to make it easier to find the file? Personal pet peeve "hey you just downloaded this file that has a logical name but we're gonna name it a UUID by default"

Reply:
Haha yeah, good point.  I rushed this out and this is a detail that i missed.  I'll fix this before merge.